main.tf

```diff
@@ -0,0 +1,67 @@
+module "s3_bucket" {
+  source = "git::https://github.com/plus3it/terraform-aws-tardigrade-s3-bucket.git?ref=5.0.0"
+
+  bucket        = var.bucket_name
+  force_destroy = true
+
+  policy = {
+    json = jsonencode({
+      "Version" : "2012-10-17",
+      "Statement" : [
+        {
+          "Effect" : "Allow",
+          "Principal" : {
+            "Service" : "acm-pca.amazonaws.com"
+          },
+          "Action" : [
+            "s3:PutObject",
+            "s3:PutObjectAcl",
+            "s3:GetBucketAcl",
+            "s3:GetBucketLocation"
+          ],
+          "Resource" : [
+            "arn:aws:s3:::${var.bucket_name}/*",
+            "arn:aws:s3:::${var.bucket_name}"
+          ],
+          "Condition" : {
+            "StringEquals" : {
+              "aws:SourceAccount" : "${data.aws_caller_identity.current.account_id}"
+            }
+          }
+        }
+      ]
+    })
+  }
+}
+
+resource "aws_acmpca_certificate_authority" "this" {
```

Comment:
There are a number of other parameters for the private ca that are also available... For example... `enabled`, `usage_mode`, `tags`, `type`, `key_storage_security_standard`, and `permanent_deletion_time_in_days`. Many of these are set in the variable object, but then are not used in the resource.